### PR TITLE
chore(osmosis-1): add $rstk

### PR DESCRIPTION
**Pull request by Sen-Com**
## Description

Adding the ReStake Token (Ticker: $RSTK) from Migaloo to the osmosis-1 asset list and zone asset list. $RSTK is the governance token of ReStake DAO which controls a fraction of Migaloo's fiscal spending. The token will be used in a stream swap and afterwards made available for trading on White Whale's Osmosis market. $RSTK will be airdropped to SAIL DAO and the Mad Scientists.

### Adding Assets

- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity
- [x] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds. The StreamSwap Bootstrapping phase begins (when?): 

_**As soon as this PR is merged.**_

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -15376,6 +15376,47 @@
         "osmosis-main",
         "osmosis-unlisted"
       ]
+    },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:15380]: Please do not update the assetlist.json file directly. This is generated only.

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -3160,6 +3160,14 @@
       "listing_date_time_utc": "2024-04-13T17:50:00Z",
       "osmosis_unlisted": true,
       "_comment": "Staked Astroport $xASTRO"
+    },
+    {
+      "chain_name": "migaloo",
+      "base_denom": "factory/migaloo1d0uma9qzcts4fzt7ml39xp44aut5k6qyjfzz4asalnecppppr3rsl52vvv/rstk",
+      "path": "transfer/channel-642/factory/migaloo1d0uma9qzcts4fzt7ml39xp44aut5k6qyjfzz4asalnecppppr3rsl52vvv/rstk",
+      "osmosis_verified": true,
+      "listing_date_time_utc": "2024-04-15T17:00:00Z",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:3169]: Should this be hidden (visible only in preview mode) until the 15th?

Comment (Sen-Com) [reply osmosis-1/osmosis.zone_assets.json:3169]: I don't really care as long as the stream swap team has what they need

File: osmosis-1/osmosis-1.assetlist.json (hunk 1 above)
Comment (Sen-Com) [reply osmosis-1/osmosis-1.assetlist.json]: done

File: osmosis-1/osmosis.zone_assets.json [hunk 3]
```diff
@@ -3160,6 +3160,14 @@
       "listing_date_time_utc": "2024-04-13T17:50:00Z",
       "osmosis_unlisted": true,
       "_comment": "Staked Astroport $xASTRO"
+    },
+    {
+      "chain_name": "migaloo",
+      "base_denom": "factory/migaloo1d0uma9qzcts4fzt7ml39xp44aut5k6qyjfzz4asalnecppppr3rsl52vvv/rstk",
+      "path": "transfer/channel-642/factory/migaloo1d0uma9qzcts4fzt7ml39xp44aut5k6qyjfzz4asalnecppppr3rsl52vvv/rstk",
+      "osmosis_verified": true,
```
[the remark this answers is not in the document]
Comment (Sen-Com) [reply osmosis-1/osmosis.zone_assets.json]: done

**Pull request merged**